pkg/mbedtls/Kconfig

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+menuconfig KCONFIG_USEMODULE_MBEDTLS
+    bool "Configure mbed TLS"
+    help
+        Configure mbed TLS using Kconfig.
+
+if KCONFIG_USEMODULE_MBEDTLS
+
+config MBEDTLS_SELF_TEST
+    bool "Enable the checkup functions (*_self_test)"
+    default y
+
+config MBEDTLS_ENTROPY_C
+    bool "Enable the platform-specific entropy code."
+    default y
+    depends on MBEDTLS_SHA256_C || MBEDTLS_SHA512_C
```

Comment:
Where is `MBEDTLS_SHA512_C` being defined?

Reply:
We do not expose it right now, a future extension could do so. I kept it here as a reminder. Furthermore, the documentation of mbedtls clearly states that "entropy" depends on sha256 or sha512

Reply:
If you really think we need to keep it there for a reference, I'd do: `depends on MBEDTLS_SHA256_C # || MBEDTLS_SHA512_C`

Reply:
done